packages/mobile/src/escrow/EscrowedPaymentListItem.tsx

```diff
@@ -21,41 +22,42 @@ interface OwnProps {
 
 type Props = OwnProps & WithNamespaces
 
-export class EscrowedPaymentReminderNotification extends React.PureComponent<Props> {
-  getCTA = () => {
+const TAG = 'EscrowedPaymentListItem'
+
+export class EscrowedPaymentListItem extends React.PureComponent<Props> {
+  onSendMessage = () => {
     const { payment, t } = this.props
     const recipientPhoneNumber = payment.recipientPhone
+    CeloAnalytics.track(CustomEventNames.clicked_escrowed_payment_send_message)
+    // TODO: open up whatsapp/text message slider with pre populated message
+    try {
+      if (Platform.OS === 'android') {
+        SendIntentAndroid.sendSms(recipientPhoneNumber, t('escrowedPaymentReminderSms'))
+      } else {
+        // TODO look into using MFMessageComposeViewController to prefill the body for iOS
+        navigateToURI(`sms:${recipientPhoneNumber}`)
+      }
+    } catch (error) {
+      Logger.showError(ErrorMessages.SMS_ERROR)
```

Comment:
Please leave a TODO for this to use the `showError` saga instead of the Logger.showError, which is a hacky temp thing we used for a while that doesn't actually work on iOS